tracer/test/Datadog.Trace.Tests/DistributedTracer/AutomaticTracerTests.cs

```diff
@@ -58,40 +58,24 @@ public void SetSpanContext()
         }
 
         [Fact]
-        public void LockSamplingPriority()
+        public void SetSamplingPriority_NoChild()
         {
-            var manualTracer = new Mock<ICommonTracer>();
-
             var automaticTracer = new AutomaticTracer();
-            automaticTracer.Register(manualTracer.Object);
-
-            ((IDistributedTracer)automaticTracer).LockSamplingPriority();
-
-            manualTracer.Verify(t => t.LockSamplingPriority(), Times.Once);
-        }
-
-        [Fact]
-        public void TrySetSamplingPriority_NoChild()
-        {
-            var automaticTracer = new AutomaticTracer();
-
-            var samplingPriority = ((IDistributedTracer)automaticTracer).TrySetSamplingPriority(SamplingPriority.UserKeep);
 
-            samplingPriority.Should().Be(SamplingPriority.UserKeep, "TrySetSamplingPriority should be pass-through when there is no child");
+            ((IDistributedTracer)automaticTracer).SetSamplingPriority(SamplingPriority.UserKeep);
```

Comment:
Are you missing an assertion here or just checking that the code doesn't throw?

Reply:
Just checking that it doesn't throw. I wish xunit had a `Assert.DoesNotThrow` to express the intent